integreat_cms/static/src/js/analytics/hix-widget.ts

```diff
@@ -2,39 +2,43 @@ import { getContent } from "../forms/tinymce-init";
 import { getCsrfToken } from "../utils/csrf-token";
 
 let initialContent: string = null;
-let initialHixValue: number = null;
+let initialHixScore: number = null;
 
-/* Display the HIX value using a bar chart */
-const updateHixBar = (value: number, setOutdated: boolean) => {
-    const roundedHixValue = Math.round(value * 100) / 100;
+/**
```

Comment:
I started to do some refactoring here to maybe improve readability a bit, because it's quite a clutter now 🙈
But decided to finish it later in order not to delay the main feature as it's high prio